components/ListView.js

```diff
@@ -67,8 +67,10 @@ class ListView extends React.Component {
   static propTypes = {
     autoHideHeader: React.PropTypes.bool,
     style: React.PropTypes.object,
-    data: React.PropTypes.array,
+    data: React.PropTypes.oneOfType([
+      React.PropTypes.object, React.PropTypes.array]),
```

Comment:
If data is object it won't work when grouping data into sections because data is grouped with reduce and object doesn't have reduce [#L41](https://github.com/Lughino/ui/blob/e0873ea5b3d6983a30585cf876ac0c9f4be95be5/components/ListView.js#L41). Update `groupItemsIntoSections` to use lodash [reduce](https://lodash.com/docs/4.16.6#reduce) (it support both object and array).